src/psyclone/tests/psyGen_test.py

```diff
@@ -2219,6 +2219,14 @@ def test_walk():
     binary_op_list = invoke.schedule.walk(BinaryOperation)
```

Comment:
It's not your fault but this test is in the wrong place. As we have a rule about not modifying things and moving them at the same time (and to make your life easier), I suggest moving the new bit into a new `test_walk_depth` test in `tests/psyir/nodes/node_test.py`. As Sergi has suggested elsewhere, you can use the `fortran_reader` test fixture to create some PSyIR from some Fortran and then test `walk` on that.

Reply:
Addressed in 53d88af1aecdb71f3734bdda20deba31cedcdea6.

Reply:
[Now https://github.com/stfc/PSyclone/pull/2377/commits/c1a4126f33df226a4559f8d762e3f0f814161f67]